source/settings/dovecot_core_settings.rst

```diff
@@ -1453,16 +1453,16 @@ Using * as the value makes Dovecot use the default value.
 
 There are currently defaults for the following fields:
 
-* ``name``
-* ``version``
-* ``os``
-* ``os-version``
-* ``support-url``
-* ``support-email``
+* ``name``: Name of distributed package (Default: ``Dovecot``)
+* ``version``: Dovecot version
+* ``os``: OS name reported by uname syscall (similar to ``uname -s`` output)
+* ``os-version``: OS version reported by uname syscall (similar to ``uname -r`` output)
+* ``support-url``: Support webpage set in dovecot distribution (Default: ``http://www.dovecot.org/``)
+* ``support-email``: Support email set in dovecot distribution (Default: ``dovecot@dovecot.org``)
 
 .. versionadded:: 2.3.10
 
-* ``revision``
+* ``revision``: Short version of dovecot source tree HEAD (same as abbreviated commit hash reported in ``dovecot --version``)
```

Comment:
Here and elsewhere: s/dovecot/Dovecot/

Reply:
And here I think also mention git. And I don't think HEAD is right, because it can be anything that where it was compiled from. Maybe just: "Short commit hash of the Dovecot git source tree (same as ..."

Also .. why use both "Short" and "abbreviated" - aren't they the same here and should use consistently the same? Maybe just leave it out? "(same as the commit hash reported ..."

Reply:
In git, HEAD *is* the checked out version :)

Reply:
Oh. I guess it can stay then :)

Reply:
changed it to "git source tree HEAD" :)